portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -2,17 +2,34 @@
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
+import com.google.gson.Gson;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    private String[] facts = {"I speak three languages","My go-to deli order is bagel with cream cheese & jelly","My Favorite book is Things Fall Apart by Chinua Achebe"};
```

Comment:
facts won't ever change, so this should be a final static variable: `private final static String[] facts = ...`